src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -331,11 +345,20 @@ export default defineComponent({
     }
 
     async function openReviewStep() {
-      currentStep.value = STEPS.review;
-      await multisigAccountPrepare(
-        confirmationsRequired.value,
-        signers.value.map((signer) => signer.address),
-      );
+      try {
+        loading.value = true;
```

Comment:
Would be nice for the user to be able to see loading state.

Reply:
added
```suggestion
        <Loader v-if="loading" />
```
in the template